components/lane_follower/src/individual_follower.py

```diff
@@ -0,0 +1,172 @@
+# Lane Follower for a single camera
+# Applies a perspective transform to an image, and isolates lane lines
+import numpy as np
+import math
+import cv2
+
+
+class IndividualFollower:
+    # Constants for how we process images
+    # These should be parametrized and better defined at a later date: as part of testing
+    NWINDOWS = 9
+    SEARCH_WIDTH = 100
+    MINPIXELS = 40
+    LANE_POLY_SIZE = 2
+
+    # Rendering Parameters
+    BOX_COLOR = (0, 255, 0)
+    LANE_COLOR = (255, 0, 0)
+
+    DRAW_THICKNESS = 2
```

Comment:
I would make a configuration dataclass so that we can easily pass them to a lane-follower. I would also give them the same defaults.

Reply:
Created a .toml